lib/index.js

```diff
@@ -1,17 +1,20 @@
 'use strict';
 
+const cssNoRTLOnly = require('./rules/cssNoRTL-only.js');
 const onlySpreadCSS = require('./rules/only-spread-css');
 const noUnusedStyles = require('./rules/no-unused-styles');
 
 module.exports = {
   rules: {
+    'cssNoRTL-only': cssNoRTLOnly,
     'only-spread-css': onlySpreadCSS,
     'no-unused-styles': noUnusedStyles,
   },
 
   configs: {
     recommended: {
       rules: {
+        'react-with-styles/cssNoRTL-only': ['error'],
```

Comment:
since we use `2` in the line below, let's change that to "error", or change this to `2`

Reply:
"error" please

Reply:
I think to this point, I noticed this and didn't change it because I couldn't remember if 2 was warning or okay.